src/org/scada_lts/service/MultiChangesHistoryService.java

```diff
@@ -0,0 +1,93 @@
+package org.scada_lts.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.jfree.util.Log;
+import org.scada_lts.dao.MultiChangesHistory;
+import org.scada_lts.dao.model.multichangehistory.MultiChangeHistoryValues;
+import org.scada_lts.service.model.MultiChangeHistoryDTO;
+import org.scada_lts.service.model.ValuesMultiChangesHistoryDTO;
+import org.scada_lts.web.mvc.api.components.cmp.model.SetValuePointDTO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * @author  grzegorz.bylica@abilit.eu on 15.10.2019
+ */
+
+@Service
+public class MultiChangesHistoryService {
+
+    @Autowired
+    private MultiChangesHistory multiChangesHistoryDAO;
+
+    public Set<MultiChangeHistoryDTO> getHistory(String idViewAndCmpId) {
+
+        List<MultiChangeHistoryValues> data = multiChangesHistoryDAO.getHistory(idViewAndCmpId);
```

Comment:
Is it possible to change name "data" to more understandable?

Reply:
improved